<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cAddLocationDialog</class>
 <widget class="QDialog" name="cAddLocationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="m_lpLocation"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="m_lpAddress"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="m_lpName"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="m_lpCountry"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Country:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Federal State:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="m_lpAddCountry">
       <property name="text">
        <string>add...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="m_lpCity"/>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="m_lpAddCity">
       <property name="text">
        <string>add...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="m_lpFederalState"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="m_lpButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_lpName</tabstop>
  <tabstop>m_lpLocation</tabstop>
  <tabstop>m_lpAddress</tabstop>
  <tabstop>m_lpCountry</tabstop>
  <tabstop>m_lpAddCountry</tabstop>
  <tabstop>m_lpFederalState</tabstop>
  <tabstop>m_lpCity</tabstop>
  <tabstop>m_lpAddCity</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>cAddLocationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lpButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>cAddLocationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
